app/services/spreadsheet_parser.rb

```diff
@@ -42,7 +42,7 @@ def self.first_two_rows(sheet)
     i = 1
     rows.each do |row_values|
       # Creek XLSX parser returns Hash of the row, Roo - Array
-      row = parse_row(row_values, sheet)
+      row = parse_row(row_values, sheet, header: i.equal?(1))
```

Comment:
oh god :)

Can you refactor this please, use `each_with_index `and don't forget to compare it with 0 instead of 1

Reply:
Refactored a bit more. Check again please :D